platform/android/java/lib/src/org/godotengine/godot/input/GodotEditText.java

```diff
@@ -122,14 +127,10 @@ private void handleMessage(final Message msg) {
 		}
 	}
 
-	private void setMaxInputLength(EditText p_edit_text, int p_max_input_length) {
-		if (p_max_input_length > 0) {
```

Comment:
I removed the conditional here because this time, if mMaxInputLength == 0, we really want the user to not be able to insert anymore character.
mMaxInputLength == 0 happens when we reach max length in LineEdit, and we move the cursor to the frontmost position of the text.